webapp/app/validators/formRules.jsx

```diff
@@ -0,0 +1,14 @@
+import { validationSettings } from './validationSettings';
+
+export const formRules = {
+  login: [
+    ...validationSettings.username,
+    ...validationSettings.password
+  ],
+  user: [
+    ...validationSettings.name,
+    ...validationSettings.email,
+    ...validationSettings.skypeId,
+    ...validationSettings.phone
+  ]
```

Comment:
I think we will need lodash cloneDeep here (unless you spread each nested object), spread will clone first level attributes. Nested Objects will be passed by reference and unexpectedly mutated.

`yarn add lodash.cloneDeep`

and here
`import cloneDeep from 'lodash.cloneDeep'`